src/transformers/trainer.py

```diff
@@ -4786,8 +4769,6 @@ def create_accelerator_and_postprocess(self):
             # take the gradient_accumulation_steps setting from TrainingArguments.
             grad_acc_kwargs["num_steps"] = self.args.gradient_accumulation_steps
 
-        grad_acc_kwargs["sync_with_dataloader"] = False
```

Comment:
If I understand correctly, since we're managing it ourselves in the trainer, would it be safer to keep it set to False?

Reply:
Yeah, let's keep it set to False !